auth.go

```diff
@@ -355,7 +355,7 @@ func (x *Central) GetTokenFromSession(sessionkey string) (*Token, error) {
 		x.Stats.IncrementInvalidSessionKey(x.Log)
 		return token, err
 	} else {
-		if time.Now().UnixNano() > token.Expires.UnixNano() {
+		if time.Now().UTC().UnixNano() > token.Expires.UnixNano() {
```

Comment:
Look at the implementation of time.UTC and time.UnixNano, and you'll see that adding "UTC()" in here has no effect, so don't do it.